definitions/divorce/json/CaseEventToFields/CaseEventToFields-nonprod.json

```diff
@@ -0,0 +1,326 @@
+[
```

Comment:
I'm actually going to remove these completely from the production version. It will make toggling easier and they're not going to be used anyway.

Reply:
done